wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Insights Annual Site Stats card

Fixes #10833 

This add the Insights Annual Site Stats card. 

**NOTE:** Since data is not yet available for this card, stub data is displayed.

To test:

---
With data:
- Go to Insights > Annual Site Stats.
- Verify the view is:

<img width="350" alt="annual" src="https://user-images.githubusercontent.com/1816888/51290052-dab12080-19bf-11e9-84a1-eddf4f70a67f.png">

---
No data:
- Since this is using fake data, you'll have to do a little hack.
  - In `SiteStatsInsightsViewModel:createAnnualSiteStatsRow` un-comment the line:
`return AnnualSiteStatsRow(totalPostsRowData: nil, totalsDataRows: nil, averagesDataRows: nil)`
- Go to Insights > Annual Site Stats.
- Verify the view is:

<img width="350" alt="annual_none" src="https://user-images.githubusercontent.com/1816888/51290213-6aef6580-19c0-11e9-8e9d-1dfdb833a808.png">
